vslib/src/SwitchStateBaseHostif.cpp

```diff
@@ -479,7 +479,7 @@ bool SwitchStateBase::hostif_create_tap_veth_forwarding(
 
     SWSS_LOG_NOTICE("interface index = %d, %s\n", sock_address.sll_ifindex, vethname.c_str());
 
-    if (ifup(vethname.c_str(), port_id, true, true))
+    if (ifup(vethname.c_str(), port_id, false, false))
```

Comment:
why do you need this line at all? you have set the admin state right before enter this function.